### ISSUE TITLE
Places block pushing Notes block down the page (anonymous users)

### ISSUE THREAD

**Issue by rjcorwin** [jywarren/plots#47]
Notes are buried when browsing as the anonymous user. http://publiclaboratory.com/tool/thermal-camera

Is that on purpose?

**Comment by jywarren** [comment]
No, well I guess nonregistered users are more likely to want to browse
horizontally rather than in a particular place... but in general were
trying to pack too much into the sidebar.
On Dec 20, 2011 10:32 PM, "R.J. Steinert (rjstatic)" <
reply@reply.github.com>
wrote:

**Comment by rjcorwin** [comment]
Hmm. Some quick prototyping with jQuery in my browser. Like it? 
<img src=http://rjsteinert.com/github/plots-1.png>

With a little work we could probably move everything in the left sidebar to a new row in the header below the listserv feed.

**Comment by jywarren** [comment]
Would moving the search and login boxes be a problem? We could also try to make them a lot more compact. 

What about truncating the file download zone, or just letting it exist on the bottom as by default?

Yeah i like the direction you're going. we need to be a bit more creative here...

**Comment by rjcorwin** [comment]
Good point. Those two blocks could definitely get a lot smaller.  I think it would make navigating and viewing content a lot more clear if the left, center, and right columns were dedicated entirely to the current content being viewed while all navigation/other functions stay in the header.  That may mean another row of buttons below Ongoing Discussions. The login and search could be drop down forms. "Post a note" may even be better suited closer to the content below it's current location in this proposed new row of navigation. Other calls to action could live there as well.
